incubator/openvpn/templates/vpn.deployment.yaml

```diff
@@ -51,9 +51,10 @@ spec:
          - |-
             ovpn_genconfig -u tcp://$EASYRSA_REQ_CN:{{ template "exposed_port" .}} \
               -n $(cat /etc/resolv.conf | grep -i nameserver|head -n1|cut -d ' ' -f2) \
-              -p "dhcp-option DOMAIN-SEARCH cluster.local" \
-              -p "dhcp-option DOMAIN-SEARCH svc.cluster.local" \
-              -p "dhcp-option DOMAIN-SEARCH {{ .Release.Namespace }}.svc.cluster.local" \
```

Comment:
What do you think about keeping both? I know that `DOMAIN` fixed it for Viscosity, but maybe other clients want `DOMAIN-SEARCH`?